### ISSUE TITLE
Change file not characterized report to use positive language

### ISSUE THREAD

**Issue by luis100** [keeps/roda#739]
Change from:
```
File is comprehensively characterized.
Missing format designation name: no.
Missing mimetype: no.
Missing PRONOM UID: no.
```

To:
```
File is comprehensively characterized.
Has format designation name: yes
Has mimetype: yes
Has PRONOM UID: yes
```

**Comment by rui-castro** [comment]
@luis100 done in PR #741

**Comment by luis100** [comment]
checked and merged into lf_dev at 06aa3ad3b89ea2dc08eb5fec212a3d69eb827a9d
